sites/shared/components/workbench/views/measies/editor.mjs

```diff
@@ -4,30 +4,59 @@ import { MeasieInput, ns as inputNs } from 'shared/components/inputs.mjs'
 import { useTranslation } from 'next-i18next'
 import { DynamicMdx } from 'shared/components/mdx/dynamic.mjs'
 
-export const ns = nsMerge('workbench', inputNs)
+export const ns = nsMerge('workbench', 'account', inputNs)
 
 export const MeasiesEditor = ({ Design, settings, update }) => {
-  const { i18n } = useTranslation(ns)
+  const { t, i18n } = useTranslation(ns)
 
   const onUpdate = (m, newVal) => {
     update.settings(['measurements', m], newVal)
   }
 
   return (
     <div className="max-w-2xl mx-auto">
-      {Design.patternConfig.measurements.map((m) => (
-        <MeasieInput
-          key={m}
-          m={m}
-          imperial={settings.units === 'umperial' ? true : false}
-          original={settings.measurements?.[m]}
-          update={(m, newVal) => onUpdate(m, newVal)}
-          id={`edit-${m}`}
-          docs={
-            <DynamicMdx language={i18n.language} slug={`docs/measurements/${m.toLowerCase()}`} />
-          }
-        />
-      ))}
+      <h5>{t('account:requiredMeasurements')}</h5>
+      {Object.keys(Design.patternConfig.measurements).length === 0 ? (
+        <p>(None)</p>
+      ) : (
+        <div>
+          {Design.patternConfig.measurements.map((m) => (
+            <MeasieInput
+              key={m}
+              m={m}
+              imperial={settings.units === 'imperial' ? true : false}
+              original={settings.measurements?.[m]}
+              update={(m, newVal) => onUpdate(m, newVal)}
+              id={`edit-${m}`}
+              docs={
+                <DynamicMdx
+                  language={i18n.language}
+                  slug={`docs/measurements/${m.toLowerCase()}`}
+                />
+              }
+            />
+          ))}
+          <br />
+        </div>
+      )}
+      <h5>{t('account:optionalMeasurements')}</h5>
+      {Object.keys(Design.patternConfig.optionalMeasurements).length === 0 ? (
+        <p>(None)</p>
```

Comment:
Don't forget about translation. Won't block the PR for this, but we should avoid hard-coding English UI output.